pkg/kubelet/cm/dra/manager.go

```diff
@@ -277,21 +278,17 @@ func (m *ManagerImpl) GetResources(pod *v1.Pod, container *v1.Container) (*Conta
 				continue
 			}
 
-			claimInfo := m.cache.get(*claimName, pod.Namespace)
-			if claimInfo == nil {
-				return nil, fmt.Errorf("unable to get resource for namespace: %s, claim: %s", pod.Namespace, *claimName)
+			claimInfo, err := m.cache.getClaimInfo(*claimName, pod.Namespace)
+			if err != nil {
+				return nil, err
```

Comment:
Why drop the extra error information?

Reply:
It's not dropped, it's moved to the `cache.getClaimInfo` since it returns error now.

Reply:
I'd still argue you need to wrap it here for consistency, especially since (by convention) it is the *caller's* responsibility to provide the namespace and claimname in the error string (not the *callee*).